### DISCUSSION TITLE
Maximum requests per minute / day / month?

### DISCUSSION THREAD

**Discussion by JackySw** [Project-OSRM/osrm-backend#6879] [General]
Hi,

I can't find numbers about maximum requests for getting the distance of two lat-lng via: https://router.project-osrm.org/route/v1/

Is 1 per second a lot? I want to retrieve about a million requests but do not want to get blocked. I can stretch it out over weeks if this is better. Or should I host it myself to avoid this?

**Comment by DennisOSRM** [comment]
One per second shouldn't be too much. If you want to compute a distance matrix you get much quicker results by researching the table request and hosting the service yourself.

**Comment by JackySw** [comment]
Thanks a lot. It is a one time operation, so I'm glad I can use the router functionality. I'll make sure it will not request too fast. If the project change and I need more than I will do a self host solution. 

But I understand there are no hard limits as long as someone does not overload the server?
